components/expenses/PayoutBankInformationForm.js

```diff
@@ -213,26 +203,44 @@ const DetailsForm = ({ disabled, getFieldName, formik, host, currency }) => {
     return <P>{error.message}</P>;
   }
 
-  const [requiredFields] = data.host.transferwise.requiredFields;
-  const [addressFields, otherFields] = partition(requiredFields.fields, f =>
+  const transactionTypeValues = data.host.transferwise.requiredFields.map(rf => ({ label: rf.title, value: rf.type }));
```

Comment:
🍜 Personal preference: I know that `rf` after `requiredField` might seem obvious as to what it is but I prefer just writing it out anyways to avoid confusion (i.e. `.map(requiredField =>...` or `.map(field =>...`)